src/EncryptionBehavior.php

```diff
@@ -340,7 +340,10 @@ protected function addEncryptionToSetter(string &$script, string $columnPhpName,
         $setterLocation = strpos($script, "set$columnPhpName");
 
         if ($setterLocation === false) {
-            throw new Exception("The 'set$columnPhpName' method name was not found in the propel model code.");
+            throw new Exception(sprintf(
+                'The "%s" method was not found in the script.',
+                "set$columnPhpName",
```

Comment:
still missing proper () as per guide

Reply:
My apologies about these millions of commits =)